docs/stdlib/array.rst

```diff
@@ -280,9 +285,7 @@ Reference
 
     :index: join array_to_string implode
 
-    Render an array to a string.
-
-    Join a string array into a single string using a specified *delimiter*:
+    Returns the elements of an array joined as a string by a delimiter.
```

Comment:
I like the previous wording here, IMO it's easier to read and understand. I've troubles parsing the new version fast.